src/reducers/__tests__/deleteEmptyThought.ts

```diff
@@ -66,23 +67,29 @@ it('do not delete thought with children', () => {
 
 })
 
-it('archive thought with hidden children', () => {
+it(`archive thought with hidden children - arvhive all children in cursor's parent`, () => {
 
   const steps = [
-    newThought(''),
-    newSubthought('=one'),
-    cursorBack,
-    deleteEmptyThought,
+    importTextReducer({
+      path: HOME_PATH,
+      text: `
+        - 
+          - =a
+          - =b`
+    }),
+
+    setCursorFirstMatch(['']),
+    deleteEmptyThought
   ]
 
   // run steps through reducer flow and export as plaintext for readable test
   const stateNew = reducerFlow(steps)(initialState())
+
   const exported = exportContext(stateNew, [HOME_TOKEN], 'text/plain')
 
   expect(exported).toBe(`- ${HOME_TOKEN}
-  - =archive
-    - ${''/* prevent trim_trailing_whitespace */}
-      - =one`)
+  - =a
+  - =b`)
```

Comment:
This doesn't look quite right. Don't we expect it to archive the children, per the original issue?

Reply:
I misunderstood the description and assumed that only the archived thoughts need to be archived in the parent, and the hidden thoughts needed to be moved to the parent. I'm sorry for getting this wrong.
It's fixed now, so please review.